crdppf/static/js/Crdppf/adminToolbar.js

```diff
@@ -1,8 +1,10 @@
 Ext.namespace('Crdppf');
 
-// create layer tree and append nodes & subnodes to it
+/***
+*create layer tree and append nodes & subnodes to it
```

Comment:
this comment is definitely wrong and was before - we're in a toolbar not a layer tree :) Probably my bad at first